test_files/comments.py

```diff
@@ -0,0 +1 @@
+"""A module for testing checking comment counting in CASTanet."""
```

Comment:
It may be a good idea to have other code (with no comments) in this file to ensure a slightly more complex Concrete Syntax Tree is created while testing.

Reply:
This will be addressed when test cases are improved upon in next sprint following @mariakimheinert's suggestions about how test cases should be structured.